### PR TITLE
Implement GItlab CI daily builds for windows platform (32 & 64)

### PR DESCRIPTION
**Pull request by nerzhul**
See https://gitlab.com/nerzhul/minetest/pipelines/8785102 as a pipeline result example

also pass build artifacts to 2 hours (some times gitlab-runners are slow)

### CODE REVIEW

**Comment by Fixer-007** [comment]
Both 64 and 32 bit builds worked for me.

**Comment by nerzhul** [comment]
Pipeline https://gitlab.com/nerzhul/minetest/pipelines/8786331 should produce a correct gettext build

File: util/buildbot/buildwin32.sh [hunk 1]
```diff
@@ -149,6 +149,10 @@ cmake .. \
 	-DLEVELDB_LIBRARY=$libdir/leveldb/lib/libleveldb.dll.a \
 	-DLEVELDB_DLL=$libdir/leveldb/bin/libleveldb.dll
 
-make package -j2
+make -j2
+
+if [ "x$NO_PACKAGE" = "x" ]; then
```
Comment (sfan5) [inline util/buildbot/buildwin32.sh]: can be shortened to
`[ "x$NO_PACKAGE" = "x" ] && make package`

File: util/buildbot/buildwin64.sh [hunk 2]
```diff
@@ -150,6 +150,10 @@ cmake .. \
 	-DLEVELDB_LIBRARY=$libdir/leveldb/lib/libleveldb.dll.a \
 	-DLEVELDB_DLL=$libdir/leveldb/bin/libleveldb.dll
 
-make package -j2
+make -j2
+
+if [ "x$NO_PACKAGE" = "x" ]; then
```
Comment (sfan5) [inline util/buildbot/buildwin64.sh]: same

**Comment by sfan5** [comment]
LGTM otherwise :+1:

**Pull request merged**